### PR TITLE
Fix misspellings & formatting in video challenges

### LINKED ISSUE #7057: Typo in Video Challenges - Computer Basics - Routers and Packets
The typo is at this link:
http://www.freecodecamp.com/videos/routers-and-packets

In the second question:
image

the word "pieces" is spelled incorrectly.

### PR DESCRIPTION
**Pull request by erictleung**
- Fix quick misspellings in video challenges
- Update capitalization formatting e.g. dev tools to Dev Tools and google to
  Google

Closes #7057.

### CODE REVIEW

File: seed/challenges/04-video-challenges/jslingo.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
       "description": [
         "We are going to cover what constitutes a variable, and the reasoning behind camelCase.",
         "A variable, also referred to as 'var', is the name or placeholder for a boolean, string, number, or other piece of static information.",
-        "You can use google dev tools to inspect the Free Code Camp home page and look for some variables.",
+        "You can use Google devtools to inspect the Free Code Camp home page and look for some variables.",
```
Comment (ltegman) [inline seed/challenges/04-video-challenges/jslingo.json]: Earlier these were referred to as `Dev Tools` so this should be changed for consistency.

Comment (erictleung) [inline seed/challenges/04-video-challenges/jslingo.json]: Whoops! I think I was fixing things to 'devtools' but then found out it is written like 'Dev Tools' and missed this one as I was making things to 'Dev Tools'. I checked the rest of the files for any leftover 'dev tools' and there shouldn't be any. Thanks for the thorough double check!

File: seed/challenges/04-video-challenges/bigonotation.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
       ],
       "tests": [
         [
-          "If an algorithm is effecient for small amounts of data, it will be equally efficient for large amounts of data.",
```
Comment (d00189906) [inline seed/challenges/04-video-challenges/bigonotation.json]: Thank

Comment (d00189906) [inline seed/challenges/04-video-challenges/bigonotation.json]: Thank

**Comment by d00189906** [comment]
Thank

**Comment by ltegman** [comment]
:+1:

**Pull request merged**